src/function/uvc.c

```diff
@@ -99,6 +99,16 @@ struct {
 		.export = usbg_set_config_node_int,		        \
 	}
 
+#define UVC_DEC64_ATTR(_name)						\
+	{								\
+		.name = #_name,						\
+		.offset = offsetof(struct usbg_f_uvc_frame_attrs, _name),     \
+		.get = usbg_get_dec,				        \
+		.set = usbg_set_dec,				        \
```

Comment:
@any1 You probably want to use the new implemented usgb_set_dec64 functions here.

Reply:
Yeah, I didn't run into this problem because I use `usbg_write_dec64` directly in `uvc_set_frame`.